central/postgres/report_pruning_test.go

```diff
@@ -82,6 +92,24 @@ func (s *ReportHistoryPruningSuite) TearDownSuite() {
 	s.testDB.Teardown(s.T())
 }
 
+func (s *ReportHistoryPruningSuite) addCollectiontoReportConfiguration(reportConfig *storage.ReportConfiguration) {
+
+	collection := storage.ResourceCollection{
+		Name: " Test Collection" + uuid.NewV4().String(),
+	}
+	err := s.collectionDS.AddCollection(s.ctx, &collection)
+	s.Require().NoError(err)
+	query := search.NewQueryBuilder().AddExactMatches(search.CollectionName, collection.GetName()).ProtoQuery()
+	collections, err := s.collectionDS.SearchCollections(s.ctx, query)
+	s.Require().NoError(err)
+	collectionID := collections[0].GetId()
```

Comment:
IIRC the `collection` object should have the ID already in it after you add to the datastore so no need to fetch again. Can you check? (same comment for the other test files)